buildmaster/old_new_porting_map.yml

```diff
@@ -0,0 +1,390 @@
+HERACOMBCCEP:
+  new_name: HERA_CC_318GEV_SIGMARED_EP
+  reference_arxiv: '1506.06042'
+  reference_hepdata: 10.17182/hepdata.68951.v1/t6
+  journal: Eur. Phys. J. C75 (2015) 580
+HERACOMBNCEM:
+  new_name: HERA_NC_318GEV_SIGMARED_EM
+  reference_arxiv: '1506.06042'
+  reference_hepdata: 10.17182/hepdata.68951.v1/t5
+  journal: Eur. Phys. J. C75 (2015) 580
+HERACOMBNCEP460:
+  new_name: HERA_NC_225GEV_SIGMARED_EP
+  reference_arxiv: '1506.06042'
+  reference_hepdata: 10.17182/hepdata.68951.v1/t4
+  journal: Eur. Phys. J. C75 (2015) 580
+HERACOMBNCEP575:
+  new_name: HERA_NC_251GEV_SIGMARED_EP
+  reference_arxiv: '1506.06042'
+  reference_hepdata: 10.17182/hepdata.68951.v1/t3
+  journal: Eur. Phys. J. C75 (2015) 580
+HERACOMBNCEP820:
+  new_name: HERA_NC_300GEV_SIGMARED_EP
+  reference_arxiv: '1506.06042'
+  reference_hepdata: 10.17182/hepdata.68951.v1/t2
+  journal: Eur. Phys. J. C75 (2015) 580
+HERACOMBNCEP920:
+  new_name: HERA_NC_318GEV_SIGMARED_EP
+  reference_arxiv: '1506.06042'
+  reference_hepdata: 10.17182/hepdata.68951.v1/t1
+  journal: Eur. Phys. J. C75 (2015) 580
+HERACOMB_SIGMARED_B:
+  new_name: HERA_NC_318GEV_EAVG_SIGMARED_BOTTOM
+  reference_arxiv: '1804.01019'
+  reference_hepdata: ''
+  journal: Eur. Phys. J. C78 (2018) 473
+HERACOMB_SIGMARED_C:
+  new_name: HERA_NC_318GEV_EAVG_SIGMARED_CHARM
+  reference_arxiv: '1804.01019'
+  reference_hepdata: ''
+  journal: Eur. Phys. J. C78 (2018) 473
+HERAF2CHARM:
+  new_name: HERA_NC_318GEV_EAVG_SIGMARED_F2
+  reference_arxiv: '1804.01019'
+  journal: Eur. Phys. J. C78 (2018) 473
+  reference_hepdata: ''
+
+## ATLAS
+# DY (Z, W)
+ATLASLOMASSDY11EXT:
+  new_name: ATLAS_Z0_7TEV_LOMASS_MLL
+  reference_arxiv: '1404.1212'
+  reference_hepdata: 10.17182/hepdata.64183.v1/t6
+  journal: JHEP 06 (2014) 112
+ATLASWRAP11CC:
+  new_name: ATLAS_WPWM_7TEV_CC
+  reference_arxiv: '1612.03016'
+  reference_hepdata: 10.17182/hepdata.76541.v1/t9
+  journal: Eur. Phys. J. C77 (2017) 367
+ATLASZRAP11CC:
+  new_name: ATLAS_Z0_7TEV_46FB_CC
+  reference_arxiv: '1612.03016'
+  journal: Eur. Phys. J. C77 (2017) 367
+ATLASWZRAP11CF:
+  new_name: ATLAS_Z0_7TEV_46FB_CF
+  reference_arxiv: '1612.03016'
+  journal: Eur. Phys. J. C77 (2017) 367
+ATLASWZRAP11CC:
+  new_name: ATLAS_DY_7TEV_46FB_CC
+  reference_arxiv: '1612.03016'
+  reference_hepdata: 10.17182/hepdata.76541.v1/t9
+ATLASZHIGHMASS49FB:
+  new_name: ATLAS_Z0_7TEV_49FB_HIMASS
+  reference_arxiv: '1305.4192'
+  reference_hepdata: 10.17182/hepdata.61422.v1/t1
+  journal: Phys.Lett. B725 (2013) 223
+ATLASWRAP36PB:
+  new_name: ATLAS_WPWM_7TEV_36PB_ETA
+  reference_arxiv: '1109.5141'
+  journal: Phys.Rev. D85 (2012) 072004
+ATLASZRAP36PB:
+  new_name: ATLAS_Z0_7TEV_36PB_ETA
+  reference_arxiv: '1109.5141'
+  journal: Phys.Rev. D85 (2012) 072004
+ATLASWZRAP36PB:
+  new_name: ATLAS_DY_7TEV_36PB_ETA
+  reference_arxiv: '1109.5141'
+ATLAS_WCHARM_WM_DIFF_7TEV:
+  new_name: ATLAS_WCHARM_7TEV_WM-LEPTONETA
+  reference_arxiv: '1402.6263'
+  journal: JHEP 1405 (2014) 068
```

Comment:
Should we even mention the distinction between the produced gauge boson and the lepton pair $\eta$? For sure, these shouldn't be part of the observable name.